src/systems/diff_drive/DiffDrive.cc

```diff
@@ -131,6 +131,12 @@ class ignition::gazebo::systems::DiffDrivePrivate
 
   /// \brief A mutex to protect the target velocity command.
   public: std::mutex mutex;
+
+  /// \brief frame_id from sdf.
+  public: std::string sdf_frame_id;
```

Comment:
nit: ign-gazebo uses camelCase for variable names. For reference, here's the [style guide](https://ignitionrobotics.org/docs/all/contributing#style-guides)